streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/AssignmentInfoTest.java

```diff
@@ -88,47 +88,63 @@ public void shouldThrowForUnknownVersion2() {
 
     @Test
     public void shouldEncodeAndDecodeVersion1() {
-        final AssignmentInfo info = new AssignmentInfo(1, activeTasks, standbyTasks, activeAssignment, standbyAssignment, 0);
-        final AssignmentInfo expectedInfo = new AssignmentInfo(1, UNKNOWN, activeTasks, standbyTasks, Collections.emptyMap(), Collections.emptyMap(), 0);
+        final AssignmentInfo info =
+            new AssignmentInfo(1, activeTasks, standbyTasks, activeAssignment, standbyAssignment, 0);
+        final AssignmentInfo expectedInfo =
+            new AssignmentInfo(1, UNKNOWN, activeTasks, standbyTasks, Collections.emptyMap(), Collections.emptyMap(), 0);
```

Comment:
Am I right in thinking that all that happened to these older ones is that you added line breaks?

Can we do that in a later PR that doesn't also change the production logic, so it's easier to see that we're not invalidating the tests here? I.e., it's suspicious to change regression tests while you're changing the logic the regression tests would be checking.

Reply:
Yeah it's just fixing the spacing, I'll remove it

Reply:
Thanks!